src/zenml/model/model_config.py

```diff
@@ -128,20 +109,38 @@ def get_or_create_model(self) -> ModelResponseModel:
                 model_name_or_id=self.name
             )
         except KeyError:
-            model_request = ModelRequestModel.parse_obj(
-                self._get_request_params(ModelRequestModel)
+            model_request = ModelRequestModel(
+                name=self.name,
+                license=self.license,
+                description=self.description,
+                audience=self.audience,
+                use_cases=self.use_cases,
+                limitations=self.limitations,
+                trade_offs=self.trade_offs,
+                ethic=self.ethic,
+                tags=self.tags,
+                user=zenml_client.active_user.id,
+                workspace=zenml_client.active_workspace.id,
             )
-            model = zenml_client.zen_store.create_model(model=model_request)
-            logger.warning(f"New model `{self.name}` was created implicitly.")
+            model_request = ModelRequestModel.parse_obj(model_request)
+            try:
+                model = zenml_client.zen_store.create_model(
+                    model=model_request
+                )
+                logger.warning(
```

Comment:
Warning sounds like implicit model creation is not recommended. Maybe info is better?

```suggestion
                logger.info(
```